packages/template-retail-react-app/app/utils/test-utils.js

```diff
@@ -132,10 +132,10 @@ export const TestProviders = ({
             return site.alias === siteAlias || site.id === appConfig['site']
         }) || getDefaultSite()
 
```

Comment:
So here we are getting the site object based on the `siteAlias`. The locale objects are attaching to the site, it'd be better logic if we found the local object based on the `site` that we found, and the `localeRef` that we pass in as props, instead of passing an object as a prop in the wrapperProps. For example:

Would the test pass be correct if I pass in something like this. Knowing that the uk site does have any locale of `it-it`? 
```
renderWithProviders(<MockedComponent />, {
        wrapperProps: {siteAlias: 'uk', locale: {id: 'it-it'}}
    })
```

Reply:
> We need to specify the `wrapperProps` because we need to write the same tests to pass two scenarios:
> 1. The Template Retail React App in the monorepo with a Multi-site, Multi-locale configuration.
> 2. A generated project using the `test-project` preset with the Singe-site, Single-locale configuration.

During testing, the values defined in `wrapperProps` overwrite the default mock configuration.
We do this to make the expected result in tests work for both the generated project and the template-retail-react-app project.